### ISSUE TITLE
Zooming is laggy/not smooth in inspection view

### ISSUE THREAD

**Issue by MichaelGoodale** [MontrealCorpusTools/ISCAN#190]
Currently the zooming functionality in inspection doesn't always work reliably making for a very laggy experience that can be frustrating.

**Comment by msonderegger** [comment]
@MichaelGoodale can you elaborate -- like doesn't work at all, or it's just slow?

**Comment by msonderegger** [comment]
looping in @mmcauliffe

**Comment by MichaelGoodale** [comment]
Oh whoops, never saw this comment. It just tends to hang for a little bit when zooming, it's still usable, it's just a little irritating, and sometimes you have to scroll more than once to get it to start zooming. Just slower and laggier than it should be on a nice computer.

**Comment by mmcauliffe** [comment]
@MichaelGoodale, did this get fixed in your previous work on the detail view?  I remember there being some comments about bug fixing.

**Comment by MichaelGoodale** [comment]
No, I fixed *some* aspects of the lagginess but I think there's just something fundamentally weird about the zooming that I can't quite figure out
